include/openssl/dh.h

```diff
@@ -65,7 +65,7 @@ extern "C" {
 DECLARE_ASN1_ITEM(DHparams)
 
 # define DH_GENERATOR_2          2
-/* #define DH_GENERATOR_3       3 */
+# define DH_GENERATOR_3          3
```

Comment:
This seems to be unused?

Reply:
yes. is that a problem?

Reply:
I don't see a need to change a public header for this.

Reply:
Okay, will revert.